### PR TITLE
Set Up Project for Service and Frontend Layers 

### PR DESCRIPTION
**Pull request by RebeccaSyria**
Sets up the project to run api routes as well as the react frontend 

To test it out locally:  
- "npm start" should run the service on localhost:5000 and the react app on localhost:3000 
- when it's running, going to http://localhost:5000/api/getCount (or any of the other api routes) should show the response (They're just dummy responses for now) 
- going to localhost:5000 without an api route should serve the static files for the frontend (you'll need to build first, this also isn't super useful for locally bc I don't think sso will work, so this will be for deployed environments) 
- going to localhost:3000 should be the react app

### CODE REVIEW

File: client/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "es5",
```
Comment (mxmeinhold) [inline client/tsconfig.json]: Not es6? Or es2020 I forget which it should be for the newer stuff. I think Devin was working on this in Brickwall. 

Not saying es5 won't work, just think we may as well go with the newer standards.

File: tsconfig.json [hunk 2]
```diff
@@ -1,19 +1,14 @@
 {
   "compilerOptions": {
+    "sourceMap": true,
     "target": "es5",
```
Comment (mxmeinhold) [inline tsconfig.json]: If you change the target in the other tsconfig I would change it here too.

**Review by mxmeinhold — APPROVED** [review]
LGTM! :ship: :it:

File: client/tsconfig.json (hunk 1 above)
Comment (RebeccaSyria) [reply client/tsconfig.json]: This is unchanged from the CSH React Boilerplate right now - I can change to es2020 if we want the newest, or es6 if we just want newer

Comment (mxmeinhold) [reply client/tsconfig.json]: Whatever you prefer

**Pull request merged**